code/modules/mob/living/simple_animal/abnormality/he/golden_false_apple.dm

```diff
@@ -293,6 +293,17 @@
 				var/mob/living/hit = target
 				if((hit.stat == DEAD) ||!ishuman(hit))//if the target is dead or not human
 					return ..()
+				if(istype(target, /mob/living/carbon/human/species/pinocchio))
```

Comment:
You could check based on faction but it might be nitpicky. Pinocchio works fine since we dont have that many carbon abnormalities.

Reply:
These only checks matter for a playable golden apple, since the hostile version doesn't use these pulses. Currently, there are no non-human opponents that would be encountered in RCA, so I don't think the faction check is necessary. These checks are run every single time golden apple performs an attack, so I would like to minimize unnecessary checks on it.

Reply:
Ah fair enough